doc/source/whatsnew/v0.23.0.txt

```diff
@@ -314,6 +314,7 @@ Removal of prior version deprecations/changes
 - The ``Panel4D`` and ``PanelND`` classes have been removed (:issue:`13776`)
 - The ``Panel``class has dropped the ``to_long``and ``toLong`` methods (:issue:`19077`)
 - The options ``display.line_with`` and ``display.height`` are removed in favor of ``display.width`` and ``display.max_rows`` respectively (:issue:`4391`, :issue:`19107`)
+- The ``labels`` attribute of the ``Categorical`` class has been drop in favor of ``codes``
```

Comment:
drop -> removed

``Categorical.labels`` in favor of  ``:attribute`Categorical.codes` ``

add the issue number, the original if you can dig it up and this one

Reply:
Thanks for the comments! There is not issue related to deprecating or removing `labels`. The commits didn't reference any, and searching in the issue tracker didn't find anything either. I guess by "this one" you mean #6581, right? I updated the PR with it.

Reply:
no I dont mean #6581 I mean the issue / PR where it was originally deprecated. Yes its a commit hash, but more useful is a specific PR.